.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,43 @@
+name: 'Feature Request'
+description: Suggest a new feature
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for tacking the time to request a new feature :pray:
+
+        This form for suggesting new feature's or enhancements for [excalidraw](https://github.com/excalidraw/excalidraw)
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of excalidraw's [Discussions tab](https://github.com/excalidraw/excalidraw/discussions)
+
+        Before submitting a feature request, please check the links below to see if there is a solution, feature request or question posted there already:
+         - excalidraw's [Discussions tab](https://github.com/excalidraw/excalidraw/discussions)
+         - excalidraw's [Open Issues](https://github.com/excalidraw/excalidraw/discussions/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
+         - excalidraw's [Closed Issues](https://github.com/excalidraw/excalidraw/discussions/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed)
+
+        The more information you fill in, the better the community can help you.
+  - type: textarea
+    id: steps
+    attributes:
+      label: Describe the feature you would like see in Excalidraw
+      description: Provide a clear and concise description.
+      placeholder: |
+        Ex. I'm always frustrated when [...]
+        Ex. I wish there was a way to do [...]
```

```suggestion
        eg. It would be great If [...]
        eg. I wish there was a way to do [...]
```
lets avoid "frustrated" in eg :)